tiup/tiup-component-cluster-scale-in.md

```diff
@@ -0,0 +1,70 @@
+---
+title: tiup cluster scale-in
+---
+
+# tiup cluster scale-in
+
+The `tiup cluster scale-in` command is used to scale in the cluster, which takes the services of the specified nodes offline, removes the specified nodes from the cluster, and deletes the remaining files from those nodes.
+
+Because the TiKV, TiFlash and TiDB Binlog components are taken to offline asynchronously (the removal needs to be executed by the API first) and the stopping process takes a long time (the command needs to keep checking whether the specified nodes have been successfully stopped), the `tiup cluster scale-in` command handles the TiKV, TiFlash and TiDB Binlog components particularly as follows:
```

Comment:
原文是

```
由于 TiKV，TiFlash 和 TiDB Binlog 组件的下线是异步的（需要先通过 API 执行移除操作）并且下线过程耗时较长（需要持续观察节点是否已经下线成功），所以对 TiKV，TiFlash 和 TiDB Binlog 组件做了特殊处理
```

意思是，如果不采取下面这种特殊方案，那么 tiup 自己就不得不一直观察节点是否下线成功，期间命令是 hang 住的，不可以退出。由于采取了下面这种方案，所以可以避免一直hang在那里观察。

Reply:
那可以改成如下这样吗？"TiUP needs to keep checking", @TomShawn, @lucklove, PTAL, thanks!
```suggestion
Because the TiKV, TiFlash, and TiDB Binlog components are taken offline asynchronously (the removal needs to be executed through the API first) and the stopping process takes a long time (TiUP needs to keep checking whether the specified nodes have been successfully stopped), the `tiup cluster scale-in` command handles the TiKV, TiFlash, and TiDB Binlog components particularly as follows:
```

Reply:
LGTM